src/lib/style/components/_fluid-modal.scss

```diff
@@ -28,11 +28,6 @@
         overflow-y: auto;
       }
       .modal-footer {
-        min-height: 45px;
-        max-height: 60px;
-        height: 6vh;
-        margin-bottom: 15px;
-
```

Comment:
The default footer height for Modals is sensible enough, we don't need to tweak it and create new problems.